Tests/scripts/download_conf_repos.sh

```diff
@@ -102,7 +102,9 @@ echo ${DEMISTO_LIC_PATH} > demisto_lic_path
 DEMISTO_PACK_SIGNATURE_UTIL_PATH="./signDirectory"
 echo ${DEMISTO_PACK_SIGNATURE_UTIL_PATH} > demisto_pack_sig_util_path
 
-clone_repository_with_fallback_branch "code.pan.run" "gitlab-ci-token" "${CI_JOB_TOKEN}" "xsoar/content-test-conf" "${CI_COMMIT_BRANCH}" 3 10 "master"
+CI_SERVER_HOST=${CI_SERVER_HOST:-code.pan.run}
```

Comment:
When the migration is over we'd need to change this default value.
(Add this to the Phase_2 issue.)